core/protos/o2control.proto

```diff
@@ -276,11 +276,17 @@ message GetRolesReply {
     repeated RoleInfo roles = 1;
 }
 
-message GetWorkflowTemplatesRequest{}
+message GetWorkflowTemplatesRequest{
```

Comment:
Will the request still work with no parameters? Right now ControlGUI makes use of this method and is not providing any parameters

Reply:
No, you can pass the following for the same behaviour.

```
repoPattern="",
revisionPattern="",
allBranches=false,
allTags=false 
```

Reply:
I could always add a wrapper of course...

Reply:
Right now we are not passing anything so as long as the request will default those values to empty is fine by me

Reply:
In `proto3` all fields are optional, and in memory a zero value is treated by Protobuf the same as an undefined field.
Thus, I would expect the following query
```
repoPattern="",
revisionPattern="",
allBranches=false,
allTags=false 
```
to be pushed by Protobuf as a message with all its fields `null`, and I'd expect the core to return a list of templates from all repos, at `HEAD` of the `master` branch.